interchaintest/chain_upgrade_test.go

```
@@ -0,0 +1,206 @@
+	}()
+)
+
+func TestBasicManifestUpgrade(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping in short mode")
+	}
+
+	t.Parallel()
+
+	// Setup chain with group-based governance
+	previousVersionGenesis := append(DefaultGenesis,
```

golangci-lint (1.62.2)

appendAssign: append result not assigned to the same slice

(gocritic)